build.gradle

```diff
@@ -12,6 +12,7 @@ plugins {
 	id 'java'
 	id 'maven-publish'
 	id 'signing'
```

Comment:
I suspect that this file still contains TAB characters.
Is CheckStyle working?

Reply:
Hi @PaulRambags

Great catch, thank you. Checkstyle is working on the directories you configure. I've updated the gradle.build file in the root directory to use only spaces.

Best regards
Daniel